src/core.plugins.js

```diff
@@ -63,6 +63,23 @@ class Plugin {
     return undefined;
   }
 
+  /**
+   * Initializes the plugin.
```

Comment:
Comments on the difference between activate and initialize. Why can't use just one? Or why not put this in ctor of plugin?